DistributionPackages/Neos.NeosIo/Configuration/NodeTypes.Disabled.yaml

```diff
@@ -5,7 +5,4 @@
   abstract: true
 
 'Neos.Neos:PluginView':
-  abstract: true
-
-'Neos.NodeTypes:Menu':
```

Comment:
Not exactly related, but we'll need the menu element to render an overview of the sub pages for the events archive